dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -115,7 +124,8 @@ private ClassDumper(
    *     API</a>
    */
   JavaClass loadJavaClass(String className) throws ClassNotFoundException {
-    return classRepository.loadClass(className);
+    String classFileName = specialClassFileLocation.getOrDefault(className, className);
```

Comment:
why are the two arguments the same?

Reply:
Added comment.

`getOrDefault(key, defaultValue)` This is shorthand of:

```
if (specialClassFileLocation.containsKey(className)) {
  classFileName = specialClassFileLocation.get(className);
} else {
  classFileName = className;
}
```

Reply:
These are the names with slashes, right? That is, https://docs.oracle.com/javase/specs/jvms/se7/html/jvms-4.html#jvms-4.2.1

Given that I'd call this a pathToClass rather than a class name (since classPath is already taken) and definitely path rather than a simple name.

Reply:
These are name with "."